frontend/amundsen_application/static/js/components/Card/index.tsx

```diff
@@ -44,7 +45,17 @@ const Card: React.FC<CardProps> = ({
         {subtitle && <h3 className="card-subtitle">{subtitle}</h3>}
       </header>
       <div className="card-body">
-        {copy && <div className="card-copy">{copy}</div>}
+        {copy && (
+          <div
+            className="card-copy"
+            role="button"
+            tabIndex={0}
+            onClick={stopEvent}
+            onKeyDown={stopEvent}
+          >
+            {copy}
+          </div>
```

Comment:
This creation of a fake button and adding a stop propagation handler to it seems a bit fishy to me.
Could we achieve the same effect by using CSS pointer-events?
Check in https://css-tricks.com/almanac/properties/p/pointer-events/

Reply:
It is something like this: https://stackoverflow.com/a/46707009

Reply:
I've tried this suggestion as well, but it didn't work either.. but I might be doing something wrong, I don't have much experience with `pointer-events`

Another option in my opinion is that eventPropagation can be stopped on the SanitizedHtml level [here](https://github.com/deliveryhero/amundsen/blob/main/frontend/amundsen_application/static/js/components/Announcements/AnnouncementsList/index.tsx#L49)

Reply:
if we stop the propagation in the "html_content" level it also works, but for me it's just another workaround and doesn't solve the problem fully